Permissionless-Software-Foundation/bch-api
# Adding a getCurrencyInfo endpoint to bch-api in both v4 and v5 routes

**Summary**
* Added a getCurrencyInfo endpoint to bch-api in both v4 and v5 routes which retrieves currency settings
* This is to allow ecash applications to parse the 'satoshisperunit' value to determine the ecash=1 full node setting, as this impacts how apps like Cashtab parses transaction history.

**Test Plan**
* start up local dev instance of bch-api via ./start-dev-example.sh
* navigate to http://localhost:3001/v4/util/getCurrencyInfo and verify response
* navigate to http://localhost:3001/v5/util/getCurrencyInfo and verify response
